backend/app/Http/Controllers/UserController.php

```diff
@@ -19,12 +19,14 @@ public function store(Request $request)
             'name' => ['required', 'unique:users', 'max:255'],
             'password' => ['required', 'confirmed'],
             'email' => ['required', 'unique:users'],
+            'isAdmin' => ['max:244']
```

Comment:
I think ther e is a validation for `Boolean` in laravel
https://laravel.com/docs/9.x/validation#rule-boolean